easybuild/easyblocks/p/petsc.py

```diff
@@ -237,10 +237,10 @@ def configure_step(self):
                 inc_spec = "-include=[%s]" % ','.join(scotch_inc)
 
                 # For some reason there is a v3 suffix added to libptscotchparmetis
-                # which is the reason for this new code.
-                req_scotch_libs = ['libesmumps.a', 'libptesmumps.a', 'libptscotch.a',
-                                   'libptscotcherr.a', 'libptscotchparmetisv3.a', 'libscotch.a',
-                                   'libscotcherr.a']
+                # which is the reason for this new code;
+                # note: order matters here, don't sort these alphabetically!
```

Comment:
but it's much easier for human eyes! :p 

lgtm

Reply:
Yeah... :)